test/ref_ops_test.cpp

```diff
@@ -7361,6 +7361,25 @@ TEST_CASE(unsqueeze_test)
     }
 }
 
+TEST_CASE(unsqueeze_dyn_test)
+{
+    migraphx::program p;
+    auto* mm = p.get_main_module();
+
+    migraphx::shape s1{migraphx::shape::float_type, {{1, 4, 0}, {3, 3, 0}, {3, 3, 0}}};
+    auto p0 = mm->add_parameter("x", s1);
+    mm->add_instruction(migraphx::make_op("unsqueeze", {{"axes", {1}}}), p0);
+    p.compile(migraphx::ref::target{});
+
+    std::vector<float> input_data(4 * 3 * 3);
+    migraphx::parameter_map params0;
+    migraphx::shape input_fixed_shape0{migraphx::shape::float_type, {4, 3, 3}};
+    params0["x"] = migraphx::argument(input_fixed_shape0, input_data.data());
```

Comment:
```suggestion
    params["x"] = migraphx::argument(input_fixed_shape0, input_data.data());
```
You can remove suffix `0` everywhere. makes me wonder if there are multiple params.

Reply:
I'll do this in a separate PR. There are a couple of tests that use multiple params.